### PR TITLE
#2509 - Web content security policy and forms updates

### PR DESCRIPTION
**Pull request by andrewsignori-aot**
## Form.io definitions fixes
- Between versions `4.19.1` and `4.19.2` of the `formiojs`, some errors started to happen in places where the "HTML Element" was used (duplicated content in the images below).
- Student Application dependents required validation needed to be adjusted.
- Validation messages where components had HTML mixed in the label content (e.g. `strong`) were displaying it in the validation message.
- `Report a change` started to fail its submission because the submitted data no longer had the input data from inside the main panel. A new panel needed to be created and components dragged into it.

### Student Welcome Page
image

#### Welcome Page
image

#### Removed `<strong>` from labels.
image

#### Update labels mixed with class and HTML
image

#### Dependents wrong validation message
image

### Institution Report a Change
image

## Non-related changes
- Added extra space between the wizard navigation and the student application content.
image
- Adjusted some `make` commands to be in sync with others (e.g. `make api`, `make web`).

### CODE REVIEW

File: sources/Makefile [hunk 1]
```diff
@@ -168,24 +168,15 @@ redis:
 	@docker compose -f docker-compose.yml up redis
 
 # Local API only
-api-only:
+api:
```
Comment (dheepak-aot) [inline sources/Makefile:171]: Please update line 95 accordingly. 

image

File: sources/packages/forms/src/form-definitions/studentwelcomepage.json [hunk 2]
```diff
@@ -140,15 +140,15 @@
                   "label": "HTML",
                   "labelWidth": "",
                   "labelMargin": "",
-                  "tag": "p",
+                  "tag": "div",
```
Comment (dheepak-aot) [inline sources/packages/forms/src/form-definitions/studentwelcomepage.json:143]: 👍

**Comment by dheepak-aot** [comment]
Thanks for the extensive testing fixes all around the form. Just one minor comment.

**Review by dheepak-aot — APPROVED** [review]
Looks good 👍

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @andrewsignori-aot

**Pull request merged**